packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -430,6 +430,69 @@ abstract class ScrollView extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// This sample code shows a scroll view that contains two [SliverList]s,
+/// allowing the scroll view to grow in both directions, by setting the
+/// [CustomScrollView.center] to the key of the second list.
+///
+/// ```dart
+/// List<int> top = [];
+/// List<int> bottom = [0];
+///
+/// @override
+/// Widget build(BuildContext context) {
+///   const Key centerKey = ValueKey('second-sliver-list');
+///   return Scaffold(
+///     appBar: AppBar(
+///       title: const Text('Press on the plus to add items above and below'),
+///       leading: IconButton(
+///         icon: const Icon(Icons.add),
+///         onPressed: () {
+///           setState(() {
+///             top.add(-top.length - 1);
+///             bottom.add(bottom.length);
+///           });
+///         },
+///       ),
+///     ),
+///     body: CustomScrollView(
+///       center: centerKey,
+///       slivers: <Widget>[
+///         SliverList(
+///           delegate: SliverChildBuilderDelegate(
+///             (BuildContext context, int index) {
+///               return Container(
+///                 alignment: Alignment.center,
+///                 color: Colors.blue[200 + top[index].hashCode % 4 * 100],
+///                 height: 100 + top[index].hashCode % 4 * 20.0,
```

Comment:
nit, remove .0 for consistency with the rest of the example

Reply:
The height needs to be a double though, and the other ints

Reply:
I'll mark it as resolved